Add per_tensor overload for quantized_layer_norm

mcremon-meta opened the pull request:
Summary: This will improve performance by removing some `full` ops

Differential Revision: D65151427

dulinriley commented on backends/cadence/reference/operators/quantized_layer_norm.cpp:
```diff
@@ -152,6 +154,48 @@ void quantized_layer_norm_out(
+        output_zero_point,
+        out);
+  } else {
+    ET_CHECK_MSG(
+        false,
+        "Unhandled input dtype %hhd",
+        static_cast<int8_t>(input.scalar_type()));
```
Use `ET_KERNEL_CHECK` instead for better error messages
```suggestion
    ET_KERNEL_CHECK(
        context,
        false,
        InvalidArgument,
        out);
```

mcremon-meta committed:
Add per_tensor overload for quantized_layer_norm (#6554)

Summary:

This will improve performance by removing some `full` ops

Reviewed By: dulinriley

Differential Revision: D65151427
